vehicles/car01.lua

```diff
@@ -0,0 +1,46 @@
+local definition = ...
+local types = {
+	"black", "blue", "brown", "cyan",
+	"dark_green", "dark_grey", "green",
+	"grey", "magenta", "orange",
+	"pink", "red", "violet",
+	"white", "yellow", "hot_rod",
+	"nyan_ride", "oerkki_bliss", "road_master"
+}
+
+for _, color in ipairs(types) do
+    if minetest.settings:get_bool("vehicle_mash.enable_" .. color .. "_car") then
+        local def = table.copy(definition)
+        local name = "car_"..color
+        local capitalization = color:gsub("^%l", string.upper)
+        local style = capitalization:gsub("_", " ")
+
+        def.description = vehicle_mash.S("@1 Car", style)
```

Comment:
This produces outputs like `Dark Green voiture` in French because the style is not translated. For a clean solution, see the comment below (or above in the review) stating the definition table idea.